x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/threat_mapping/create_threat_signals.ts

```diff
@@ -213,8 +213,24 @@ export const createThreatSignals = async ({
       }
       ruleExecutionLogger.debug(`Documents items left to check are ${documentCount}`);
 
+      const sortIds = getSafeSortIds(list.hits.hits[list.hits.hits.length - 1].sort);
+
+      // ES can return negative sort id for date field, when sort order set to desc
```

Comment:
Do we know the reason why ES returns negative value?

Reply:
when value in sorted field is absent, ES would return max integer 9223372036854776000

If sort is asc it would positive number: 9223372036854776000
if sort is desc it would be negative: -9223372036854776000